apps/ios/src/Podfile

```diff
@@ -29,6 +29,7 @@ use_test_app! do |target|
   target.app do
 
     pod 'FluentUI-React-Native-Shimmer', :path => '../../../packages/components/Shimmer/FluentUIReactNativeShimmer.podspec'
+    pod 'RNCPicker', :path => '../../../node_modules/@react-native-community/picker/RNCPicker.podspec'
```

Comment:
note: We really shouldn't need to do this and I need to figure out why it isn't just auto adding native modules.